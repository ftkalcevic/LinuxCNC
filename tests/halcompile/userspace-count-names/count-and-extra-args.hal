loadusr -W userspace_count_names count=2 zip
list pin
unloadusr userspace_count_names
waitusr userspace_count_names

loadusr -W userspace_count_names zip count=2 zap
list pin
unloadusr userspace_count_names
waitusr userspace_count_names

loadusr -W userspace_count_names  zap zoot count=2
list pin
unloadusr userspace_count_names
waitusr userspace_count_names

