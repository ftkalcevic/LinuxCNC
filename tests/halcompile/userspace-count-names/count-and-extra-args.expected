count=2
argc=2
argv[0]=userspace_count_names
argv[1]=zip
argv[2]=(null)
userspace-count-names.0.out
userspace-count-names.1.out

count=2
argc=3
argv[0]=userspace_count_names
argv[1]=zip
argv[2]=zap
argv[3]=(null)
userspace-count-names.0.out
userspace-count-names.1.out

count=2
argc=3
argv[0]=userspace_count_names
argv[1]=zap
argv[2]=zoot
argv[3]=(null)
userspace-count-names.0.out
userspace-count-names.1.out

